packages/web3torrent/src/components/domain-budget/DomainBudget.tsx

```diff
@@ -100,6 +101,41 @@ export const DomainBudget: React.FC<DomainBudgetProps> = props => {
     <table>
       <tbody>
         <tr>
+          <td>
+            <p>
+              <Tooltip
+                title="A budget represents the funds that web3torrent is managing on your behalf. These funds can
+          be used to pay for parts of a file."
+              >
+                <HelpOutlineIcon style={{color: '#ea692b', fontSize: 20}} />
```

Comment:
Removes header link to budgets page (page remains so you can navigate manually if you know the trick). 

Layout is not perfect, but possibly acceptable. We could ad more tooltips to the progress bars?

![Screenshot 2020-06-18 at 15 48 59](https://user-images.githubusercontent.com/1833419/85035716-66157f00-b17b-11ea-9f28-997f139bf544.png)
![Screenshot 2020-06-18 at 15 49 04](https://user-images.githubusercontent.com/1833419/85035721-6746ac00-b17b-11ea-9203-fd4f617212a3.png)